src/tribol/physics/CommonPlane.cpp

```diff
@@ -410,14 +406,12 @@ int ApplyNormal< COMMON_PLANE, PENALTY >( CouplingScheme const * cs )
         const real nodal_force_y2 = force_y * phi2[a];
         const real nodal_force_z2 = force_z * phi2[a];
 
-        #ifdef TRIBOL_DEBUG_LOG
-           dbg_sum_force1 += magnitude( nodal_force_x1, 
-                                        nodal_force_y1, 
-                                        nodal_force_z1 );
-           dbg_sum_force2 += magnitude( nodal_force_x2,
-                                        nodal_force_y2,
-                                        nodal_force_z2 );
-        #endif /* TRIBOL_DEBUG_LOG */
+        dbg_sum_force1 += magnitude( nodal_force_x1, 
+                                     nodal_force_y1, 
+                                     nodal_force_z1 );
+        dbg_sum_force2 += magnitude( nodal_force_x2,
+                                     nodal_force_y2,
+                                     nodal_force_z2 );
```

Comment:
Do you expect this to have a noticeable affect on performance? If so, it's probably be best to only compute it in debug configs as it was before.

Reply:
I removed the `TRIBOL_DEBUG_LOG` which guarded these calculations. It isn't perfect, but I added an if statement around these querying the coupling scheme logging level and only compute these if the logging level is DEBUG. Can refactor this with ifdef later?